OpenShift/init.sh

```diff
@@ -41,30 +41,28 @@ echo
 echo "Configuring project permissions..."
 echo "=================================="
 echo
-# Grant Default CI Account Edit Access to All Projects and OpenShift Project
+# Grant Default Nexus Account Edit Access to All Projects and OpenShift Project
 oc policy add-role-to-user edit system:serviceaccount:${OPENSHIFT_NEXUS_PROJECT}:default -n ${OPENSHIFT_NEXUS_PROJECT}
```

Comment:
TBH I think this is leftover copy pasta from a solution we saw on GitHub. Looking through the commit history it seems it was copied from a CI script and I see references to it in the Jenkins tutorial so I'm guessing that's where it came from.

Anyhow IIUC this can be removed as the use case for Jenkins does not hold here.